dev/io.openliberty.security.jakartasec.3.0.internal_fat.common/src/io/openliberty/security/jakartasec/fat/utils/MessageConstants.java

```diff
@@ -43,6 +43,8 @@ public class MessageConstants extends com.ibm.ws.security.fat.common.MessageCons
     public static final String CWWKS2407E_ERROR_VERIFYING_RESPONSE = "CWWKS2407E";
 
     public static final String CWWKS2410E_CANNOT_FIND_STATE = "CWWKS2410E";
+    public static final String CWWKS2411E_STATE_VALUE_IN_CALLBACK_DOES_NOT_MATCH_STORED_VALUE = "CWWKS2411E";
+    public static final String CWWKS2413E_CALLBACK_URL_DOES_NOT_MATCH_REDIRECT_URI = "CWWKS2413E";
```

Comment:
Since the code should not do a Tr.error for the CWWKS2411E message, the tests would need to be modified to not expect it.